src/test/java/org/isf/patient/test/Tests.java

```diff
@@ -643,9 +643,9 @@ public void testPatientOpdConstructor() throws Exception {
 		assertThat(patient.getCode()).isNull();
 		assertThat(patient.getBirthDate()).isNull();
 
-		assertThat(patient.getDeleted()).isEqualTo("N");
-		patient.setDeleted("Y");
-		assertThat(patient.getDeleted()).isEqualTo("Y");
+		assertThat(patient.getActive()).isEqualTo(Integer.valueOf(0));
+		patient.setActive(Integer.valueOf(1));
+		assertThat(patient.getActive()).isEqualTo(Integer.valueOf(1));
```

Comment:
I think it's the opposite here (deleted = "N" means active = 1)

I don't know the usefulness of this test though...